aws_cloudwatch_policies/aws_cloudwatch_sensitive_loggroup_encrypted.yml

```diff
@@ -0,0 +1,88 @@
+AnalysisType: policy
+Filename: aws_cloudwatch_sensitive_loggroup_encrypted.py
+PolicyID: AWS.CloudWatchLogs.SensitiveLogGroup.Encryption
+DisplayName: Sensitive AWS CloudWatch Log Encryption
+Enabled: false
+ResourceTypes:
+  - AWS.CloudWatch.LogGroup
+Tags:
+  - AWS
+  - Configuration Required
+  - Panther
+Severity: High
+Description: >
+  AWS automatically performs server-side encryption of logs, but you can encrypt with your own CMK
+  to protect extra sensitive log data.
+Runbook: >
+  Encrypt the CloudWatch log group with a KMS key, or add this log group to the ignore list.
+Reference: https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/encrypt-log-data-kms.html
+Tests:
+  -
+    Name: Logs Are Encrypted
+    ExpectedResult: true
+    Resource:
+      {
+        "ARN": "arn:aws:logs:us-west-2:1234456789012:log-group:LogGroup-2",
+        "AccountID": "123456789012",
+        "Arn": "arn:aws:logs:us-west-2:1234456789012:log-group:LogGroup-2",
+        "CreationTime": 1234567890123,
+        "KmsKeyId": "arn:aws:kms:us-west-2:123456789012:key/12345678-1234-1234-1234-123456789012",
+        "LogGroupName": "LogGroup-2",
```

Comment:
I think it helps novice readers to understand if we emphasize the attributes of events that are (and are not) relevant. In this case, you could say `"LogGroupName": "AnyLogGroup"` to emphasize that the logic cuts off before even evaluating the name, if it's encrypted.